projects/assets/src/translations/en/common.ts

```diff
@@ -120,11 +120,6 @@ export const common = {
     total: '{{total}}',
     count: '{{count}}',
   },
-  // deprecated since 3.2, moved to @spartacus/user/account/assets
-  miniLogin: {
-    userGreeting: 'Hi, {{name}}',
-    signInRegister: 'Sign In / Register',
-  },
```

Comment:
List all translation keys that were moved to other libs in the migration doc.

Reply:
done